.github/workflows/detect-breaking-changes-levitate.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           path: './pr'
       - uses: actions/setup-node@v4
         with:
-          node-version: 20.9.0
+          node-version: 22.11.0
```

Comment:
As we have the .nvmrc can we replace usage of `node-version` with `node-version-file`? [Docs link](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file).

Reply:
hmmmmmm, seems like it can't find the file in CI now 😕 any ideas @jackw before i just revert it to using the `node-version` to be safe? 🤔